src/components/InstructorRevenueCard/index.js

```diff
@@ -1,15 +1,20 @@
 import React from 'react'
 import {find, size, map} from 'lodash'
-import {Card, Button, Heading, Maybe} from 'egghead-ui'
+import {Card, Maybe} from 'egghead-ui'
 import {Text} from 'react-localize'
 import WrappedRequest from 'components/WrappedRequest'
 import OpenToggle from 'components/OpenToggle'
 import currentMonthStartDate from './utils/currentMonthStartDate'
 import totalRevenue from './utils/totalRevenue'
 import removeRevenueMonth from './utils/removeRevenueMonth'
+import prettyMonthName from './utils/prettyMonthName'
 import RevenuePeriod from './components/RevenuePeriod'
 import LineChart from './components/LineChart'
 
+const revenueColor = '#59cd90' // # blue
+const minutesColor = '#B0B6BE' // # dark-gray-secondary
```

Comment:
Will be able to replace these hard coded colors here once https://github.com/eggheadio/tachyons-egghead/issues/28 is done